config/tests/samples/create/harness.go

```diff
@@ -483,6 +483,9 @@ func NewHarness(ctx context.Context, t *testing.T) *Harness {
 		}
 	}()
 
+	// Given the informers a chance to sync
+	mgr.GetCache().WaitForCacheSync(ctx)
```

Comment:
If we need to wait for the cache anyway, would it make sense to wait for the cache to be synced and then run the tests?